cmd/manager/main.go

```diff
@@ -387,4 +388,26 @@ func setupWebhook(mgr manager.Manager, certRotation certificates.RotationParams,
 		log.Error(err, "unable to create webhook", "webhook", "Elasticsearch")
 		os.Exit(1)
 	}
+
+	// wait for the secret to be populated in the local filesystem before returning
+	interval := time.Second * 1
+	timeout := time.Second * 30
+	keyPath := path.Join(mgr.GetWebhookServer().CertDir, "tls.crt")
+	err := wait.PollImmediate(interval, timeout, func() (bool, error) {
+		_, err := os.Stat(keyPath)
+		// err could be that the file does not exist, but also that permission was denied or something else
+		if os.IsNotExist(err) {
+			log.V(1).Info("Webhook certificate file not present on filesystem yet", "path", keyPath)
```

Comment:
Maybe we should log something at the default level at least one time. Just to let the user/support knows if the operator is stuck here ?